presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -341,6 +342,10 @@ public static Optional<CompressionCodec> getCompressionCodec(TextInputFormat inp
 
     public static InputFormat<?, ?> getInputFormat(Configuration configuration, String inputFormatName, boolean symlinkTarget)
     {
+        AlphaInputFormat alphaInputFormat = null;
```

Comment:
I am deliberately throwing the exception to check if ORC query and Alpha query is throwing this exception. ORC query is throwing this exception but alpha query is not